### ISSUE TITLE
Links on Testimonial Slider fail colour contrast

### ISSUE THREAD

**Issue by mmafe** [ccswbs/ugnext#210]
DARC team feedback: "Blue on light grey failed regular text."
<img width="1524" height="488" alt="Image" src="https://github.com/user-attachments/assets/57935fe6-b9fb-40df-be31-cab10499b923" />
